### PR TITLE
ADEN Add verify environment

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java [hunk 1]
```diff
@@ -107,6 +107,8 @@ public static Credentials getCredentials() {
   public static String getEnvType() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java]: MAJOR The Cyclomatic Complexity of this method "getEnvType" is 11 which is greater than 10 authorized. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MAJOR 5 major
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Move this closing curly brace to the next line.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L22) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L36) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L24) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L25) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L37) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L33) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by nandy-andy** [comment]
LGTM :+1:

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MAJOR 5 major
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L24) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Move this closing curly brace to the next line.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L22) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L25) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L33) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L36) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/4e3f3eef82272170a8b84542d621fef304eede51/src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java#L37) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)

**Pull request merged**